index.js

```diff
@@ -34,9 +32,17 @@ function structuredClone(input, memory) {
     if (input instanceof Boolean || input instanceof Number || input instanceof String || input instanceof Date) {
         output = new input.constructor(input.valueOf());
     } else if (input instanceof RegExp) {
-        output = new RegExp(input.source, "g".substr(0, Number(input.global)) + "i".substr(0, Number(input.ignoreCase)) + "m".substr(0, Number(input.multiline)));
-
-        // Supposed to also handle Blob, FileList, ImageData, ImageBitmap, ArrayBuffer, and "object with a [[DataView]] internal slot", but fuck it
+        output = new RegExp(input.source, input.flags);
```

Comment:
Oh, I think I did it that way because your way only works in Node 6+.... but that's fine for now, Node 6 is old enough :)